api/query.go

```diff
@@ -240,6 +248,8 @@ func (e *Endpoint) SlicePost(ctx *gin.Context) {
 // @Accept   application/json
 // @Produce  multipart/mixed
 // @Success  200 {object} vds.FenceMetadata "(Example below only for metadata part)"
+// @Failure  400 {object} ErrorResponse "Request is invalid"
+// @Failure  500 {object} ErrorResponse "openvds failed to process the request"
```

Comment:
Nothing is ever perfect when it comes to docs. Both of these render with the example value from ErrorResponse,  which looks kinda weirds, as the description ("Request is invalid") is not visible in the same view. Can we inline a custom error on 400?

Reply:
Yeah, I think a more generic error is better. Maybe even `{ "error": "message" }`